components/partials/fee-discounts/overview.vue

```diff
@@ -1,92 +1,87 @@
+            <div class="flex justify-start gap-6 lg:gap-8">
+              <div class="flex flex-col">
+                <span class="text-gray-500 uppercase tracking-wide text-xs mb-2 font-semibold whitespace-nowrap">
+                  {{ $t('fee_discounts.my_staked_amount') }}
+                </span>
+                <span class="uppercase text-xs lg:text-base text-gray-500 font-bold tracking-widest whitespace-nowrap">
+                  <b class="text-xl lg:text-2xl font-bold text-white tracking-normal font-mono">{{ stakedAmount }}</b> INJ
+                </span>
               </div>
-            </template>
-          </VItem>
-          <VItem class="col-span-2 lg:col-span-3">
-            <template slot="value">
-              <span
-                v-if="feeDiscountAccountInfo"
-                class="font-mono text-lg leading-5"
-              >
-                % {{ makerFeeDiscount }} / {{ takerFeeDiscount }}
+            </div>
+            <div class="mt-4">
+              <span class="text-xs text-gray-400">
+                {{ $t('fee_discounts.current_apr') }}: {{ aprToFormat }}%
               </span>
-              <span v-else class="text-xs text-gray-400 font-mono"></span>
-            </template>
-            <template slot="title">
-              <div class="flex items-center justify-center">
-                {{ $t('trade.myMakerTakerDiscount') }}
-                <IconInfoTooltip
-                  class="ml-2"
-                  :tooltip="$t('trade.myMakerTakerDiscountTooltip')"
-                />
+            </div>
+          </div>
+        </div>
+        <div class="fee-discounts-statistic bg-gray-800 rounded-lg md:col-span-3 p-6">
+            <div class="flex flex-col">
+            <div class="flex justify-start gap-6 lg:gap-8">
+              <div class="flex flex-col">
+                <span class="text-gray-500 uppercase tracking-wide text-xs mb-2 font-semibold whitespace-nowrap">
+                  {{ $t('fee_discounts.my_trading_volume') }}
+                </span>
+                <span class="uppercase text-xs lg:text-base text-gray-500 font-bold tracking-widest whitespace-nowrap">
+                  <b class="text-xl lg:text-2xl font-bold text-white tracking-normal font-mono">{{ volumeToFormat }}</b> USD
+                </span>
               </div>
-            </template>
-          </VItem>
+            </div>
+            <div class="mt-4">
+              <span class="text-xs text-gray-400">{{ $t('fee_discounts.in_past_days', { days: daysPassed }) }}</span>
+            </div>
+          </div>
         </div>
       </div>
-      <UserWalletConnectWarning v-else cta />
-    </HocLoading>
-  </VCard>
+      <div class="flex gap-2 justify-center mt-4 md:hidden">
+        <div
+          v-for="i in 3"
+          :key="`fee-discount-statistic-slide-${i}`"
+          class="w-2 h-2 rounded-full bg-white"
+          :class="slideIndex === i - 1 ? 'bg-opacity-50' : 'bg-opacity-10'"
```

Comment:
okay, this is fine but in general when something like this needs to happen it's better to extract it to component that has a computed property, it's much cleaner